src/main/java/com/example/bookstoreapp/entity/Book.java

```diff
@@ -0,0 +1,36 @@
+package com.example.bookstoreapp.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@Data
```

Comment:
Using Data with Entity is not recommended

Reply:
Why?